src/frontend/src/components/base/Range/Range.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { tv } from "tailwind-variants";
+
+export const rangeStyles = tv({
+    slots: {
```

Comment:
The slots here have similar style definitions as those in Meter. However, I separated them considering styling changes and tweaks down the road.